src/main/java/core/basesyntax/SupplierReport.java

```diff
@@ -1,25 +1,28 @@
 package core.basesyntax;
 
 public class SupplierReport {
-    private static final String SUPPLY = "supply";
-    private static final String BUY = "buy";
-    private static final String RESULT = "result";
-    private final StringBuilder toWrite = new StringBuilder();
-
     public String creatReport(String data) {
-        String[] readiedData = (data).split(" ");
+        final String comaSeparator = ",";
+        final String spaceSeparator = " ";
+        final String supply = "supply";
+        final String buy = "buy";
+        final String result = "result";
+        final int inexOfAmount = 1;
```

Comment:
you should've only move `StringBuilder` into method :) lets move all these into fields and make them constant